### PR TITLE
Polling unit

### PR DESCRIPTION
**Pull request by bioye**
Just implemented a React version of pollingunits.html as pollingunitsreact.html using webpack.

### CODE REVIEW

File: src/main/resources/templates/pollingunits.html [hunk 1]
```diff
@@ -13,7 +21,7 @@
 	<body>
     <h2>Polling Units</h2>
     <div>
-    <table>
+    <table class="striped responsive-table">
```
Comment (bioye) [inline src/main/resources/templates/pollingunits.html]: @dinobi @benfluleck This html page is the static version that displays a table of records that is pageable and sortable.  Deployed at https://naijagov.herokuapp.com/pollingunits

File: src/main/resources/templates/pollingunitsreact.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            <script type="text/javascript" th:src="@{/js/googleanalytics.js}"></script>
+            </head>
+            <!--Start of Tawk.to Script-->
+            <script type="text/javascript" th:src="@{/js/tawkto.js}"></script>
+            <!--End of Tawk.to Script-->
+            <!--JavaScript at end of body for optimized loading-->
+            <script type="text/javascript" th:src="@{/js/materialize.min.js}"></script>
+
+            <!-- Load our React component. -->
+            <script th:src="@{/built/bundle.js}"></script>
```
Comment (bioye) [inline src/main/resources/templates/pollingunitsreact.html]: @dinobi @benfluleck This html page is the basic react version that displays a table of records. Not yet pageable and sortable.  Goal is to also introduce full-text search and column filtering.  Deployed at https://naijagov.herokuapp.com/pollingunitsreact

File: src/main/resources/static/js/pollingunits.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const React = require('react');
```
Comment (bioye) [inline src/main/resources/static/js/pollingunits.js]: Main file where the JS logic is implemented. @benfluleck @dinobi

File: src/main/resources/static/js/pollingunits.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+}
+class PollingUnits extends React.Component{
+	render() {
+		//if (this.props.data) {
+			const pollingunits = this.props.pollingunits.map(pollingunit =>
+				<PollingUnit key={pollingunit._links.self.href} pollingunit={pollingunit}/>
+			);
+		//}
+		return (
+			<table>
```
Comment (sfagade) [inline src/main/resources/static/js/pollingunits.js]: You might want to take a look at your text editor, the indentation here is not properly formed

Comment (bioye) [reply src/main/resources/static/js/pollingunits.js]: @sfagade this is actually a long line that spilled to another line.  Also, the commented lines have an effect.  Should be OK in the next PR.  Thanks.

**Pull request merged**